comfy_cli/command/distribution.py

```
@@ -1146,3 +1154,129 @@ def delete_cmd(
+):
+    renderer = get_renderer()
+    client = _builder_client(renderer, builder_url)
+    signed = _builder_call(renderer, lambda: client.get_artifact_download(artifact_id))
+    if renderer.is_pretty():
+        renderer.print(signed.get("downloadUrl", "(no url)"))
+    renderer.emit(signed, command="distribution artifact download")
+
+
+@blob_app.command("list", help="List the workspace's private blobs.")
+@tracking.track_command("distribution")
+def blob_list(
```

Pylint (4.0.6)

[convention] Missing function or method docstring

(C0116)